charts/datadog/templates/_daemonset-volumes-linux.yaml

```diff
@@ -165,8 +165,11 @@
   name: logdockercontainerpath
 {{- end }}
 {{- end }}
+{{- if .Values.datadog.containerRuntimeSupport.enabled }}
```

Comment:
I think that the same `if` statement should be put around the `runtimesocket` and `containerdsocket` Windows volumes there: https://github.com/DataDog/helm-charts/blob/40668755ff79999cf9085b01184c8d9849e71ed6/charts/datadog/templates/_daemonset-volumes-windows.yaml#L24-L34

Reply:
done